packages/scss/src/theming/components/_textfield.theme.scss

```diff
@@ -1,93 +1,94 @@
 $textfield: (
-  font-size: _theme("sizes.standard.font-size"),
-  background: #FFFFFF,
-  border-radius: .2rem,
+	font-size: _theme("sizes.standard.font-size"),
+	background: #ffffff,
```

Comment:
This whole file feels like there has been a prettier going through :D Colors aren't uppercase anymore, there are leading zeros on dimensions ( 0.2rem instead of .2rem) and so on.

Reply:
✔️